oximeter/db/src/client.rs

```diff
@@ -3145,12 +3145,18 @@ mod tests {
         serde_json::Value: From<T>,
     {
         let datum = Datum::from(hist);
-        let measurement = Measurement::new(Utc::now(), datum);
+
+        // We artificially give different timestamps to avoid a test flake in
+        // CI (reproducible reliably on macOS) where the two Utc::now() are the
+        // same, which means we get two results on retrieval when we expect one
+        let t1 = Utc::now();
+        let t2 = t1 + Duration::from_nanos(1);
```

Comment:
So this particular test structure, inserting a real and missing sample, occurs in many places. Is it worth it to make this a method or something, and then call that in the other `*_test_impl` functions?

Reply:
I'll take a look. What is odd is that those don't fail for me locally even though in many cases they seem to be using not two `Utc::now()` calls but literally the same timestamp. So I might need your help understanding what that's about.

Reply:
Ok, I think I understand why that's the case. This particular test of histograms works a bit differently from the others, which tests scalars. Those construct a sample, run the test for recalling it, and then create the missing sample and run that recall test. This one constructs both samples back-to-back, and then runs each recall test.

So I think the fix you have here is fine, since this one test is the only test where they're constructed back-to-back.